test/features/deploy_travel_agency.feature

```diff
@@ -52,7 +54,32 @@ Feature: Deploy Travel agency service and verify its functionality
 
     Then Service "travels" with process name "travels" is available
 
-  Scenario: Travel application with required Visa
+	Examples:
```

Comment:
indentation :D

Reply:
Fixed. I've already configured my VS to avoid this issue in the future.